test/support/fulcio_helper.rb

```diff
@@ -0,0 +1,86 @@
+module FulcioHelper
+  include UrlHelper
+  include SigstoreAuthHelper
+
+  FULCIO_BASE_URL = 'https://fulcio.sigstore.dev/'
+
+  def fulcio_api_url(*path, **kwargs)
+    url_regex(FULCIO_BASE_URL, 'api', 'v1', path, **kwargs)
+  end
+
+  def fulcio_signing_cert_url
+    fulcio_api_url('signingCert')
+  end
+
+  def stub_sigstore_auth_get_openid_config(returning: {})
```

Comment:
This method probably belongs to the SigstoreAuthHelper, right?

Reply:
Oops, yes it does.